exchange/bidder_test.go

```diff
@@ -3056,3 +3056,91 @@ func TestGetBidType(t *testing.T) {
 		})
 	}
 }
+
+func TestSeatNonBid(t *testing.T) {
```

Comment:
Are there any other test cases you can think of that would make sense here? I only ask since I see only one test case and was wondering if there are any others that would make sense.